templates/SilverStripe/Blog/Includes/BlogSideBar.ss

```diff
@@ -0,0 +1,38 @@
+<nav role="navigation" class="page-sidebar-widget page-sidebar-nav">
+  <p class="text-center"><span class="font-icon-rss"></span><a href="$BlogPage.Link('rss')">Subscribe to RSS</a></p>
```

Comment:
I'm not seeing the icon. Can we make "Subscribe to RSS" translatable.

Reply:
There is no icon. Have removed.